src/ErrorHandler.php

```diff
@@ -29,20 +29,20 @@ public function onError($code, $message, $file, $line)
         switch ($code) {
             case E_NOTICE:
             case E_USER_NOTICE:
-                $exc = new Errors\Notice($message, $file, $line, debug_backtrace());
+                $exc = new Errors\Notice($message, '', 0, debug_backtrace());
```

Comment:
So, if I understand this correctly, when an uncaught error is thrown, this code is invoked and it builds an exception for Airbrake. And when that happens, PHP doesn't provide an exception object, only, the info provided as parameters of this function, so exception building is our responsibility.

If that is true, it seems like it never provides `$file` & `$line`, since you just hardcoded them. I think I am confused now :-P

Why is this hardcoded if the `onError` function accepts `$file` & `$line`?